src/env-inl.h

```diff
@@ -40,6 +40,31 @@
 
 namespace node {
 
+namespace loader {
+inline PackageConfig::PackageConfig(PackageConfig&& pkg)
+    : exists(pkg.exists),
+      is_valid(pkg.is_valid),
+      has_main(pkg.has_main),
+      main(pkg.main),
+      type(pkg.type),
+      has_exports(pkg.has_exports),
+      exports((v8::Global<v8::Value>&&) pkg.exports) {
```

Comment:
style nit: We avoid C-style casts where we can, i.e.

```suggestion
      exports(std::move(pkg.exports)) {
```